cms/models.py

```diff
@@ -240,11 +240,45 @@ class CoursesInProgramPage(CourseProgramChildPage):
         blank=True,
         null=True,
     )
+    override_contents = models.BooleanField(
+        blank=True,
+        default=False,
+        help_text="Manually select contents below. Otherwise displays all courses related associated with the program.",
```

Comment:
Thanks but actually this text is confusing me, It seems me it will only override courses related associated with the program. I am thinking it will also show up programs too not just the courses in programs. plz correct me over there or update the help text accordingly.

Reply:
@asadiqbal08 This carousel can only be added as a child of a `ProgramPage` or the `HomePage`.

To clarify, it allows you to select **contents** from the CMS. Regardless of which pages you select underneath (courses only/courses and programs/programs only) they will show up in the carousel.

On a `HomePage` because there is no associated `Program` the carousel would normally show empty. That's where the override comes in.

On the flip side, if this is under a `ProgramPage` the carousel would by default show all the courses in the associated program. Even then, you could check `override_contents` and select which contents you wanted to show.

In summary, the override actually overrides all default content and shows only what you select within the CMS.

@pdpinch could you please pitch in and also advise if the help text needs to be updated and to what.